src/main/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/clients/CcdCaseAssignment.java

```diff
@@ -155,25 +136,13 @@ public void applyNoc(
                 setHeaders(serviceAuthorizationToken, accessToken)
             );
 
-        ResponseEntity<Object> response;
-        try {
-            response = restTemplate
-                .exchange(
-                    aacUrl + applyNocAssignmentsApiPath,
-                    HttpMethod.POST,
-                    requestEntity,
-                    Object.class
-                );
-
-        } catch (RestClientResponseException e) {
-            throw new CcdDataIntegrationException(
-                "Couldn't apply noc AAC case assignment for case ["
-                + callback.getCaseDetails().getId()
```

Comment:
the old logging matches ccd id with the problem, I am not sure if the new logging supports this, can you check?